user-guide/live-traffic/live-traffic.md

```diff
@@ -127,32 +144,36 @@ Many operations begin by selecting one or more entries in the Live Traffic and t
 
 When the Inspectors are activated, they will automatically decide which Inspector is best suited to display the selected session's request and response.
 
-### Live Traffic Columns
+## Live Traffic Columns
 
-Certain key information is displayed in the columns of the Live Traffic, including:
+Certain key information is displayed in the columns of Live Traffic, including:
 
 - __\#__ - An identification number generated by Fiddler Everywhere
 - __Result__ - The status code from the response
 - __Protocol__ - The protocol (HTTP/HTTPS/FTP) used by the Session
 - __Host__ - The hostname and the port of the server to which the request was sent
 - __URL__ - The URL path, file, and query string from the request
-- __Method__ - The HTTP Method used by the request (GET, POST, etc.)
+- __Method__ - The HTTP Method used by the made request (GET, POST, PUT, etc.)
 - __Body__ - The number of bytes in the response body
 - __Caching__ - Values from the Response's Expires and Cache-Control headers
 - __Content-Type__ The Content-Type header from the response
 - __Process__ The local OS process from which the traffic originated
-- __Comments__ Field for showing the custom comments added by you (or the author of a shared session).
+- __Comments__ A field for showing the custom comments added by you (or the author of a shared session).
+
+Each column comes with a **more** button (three vertical dots), which opens a **Filter** popup. Use this functionality to apply filters for the currently selected column only (e.g., filter all the traffic by _host_ name or by a specific _status code_). Columns with active filters will have a blue filter indicator. You can clear active filters via the **Filter** popup (for the filtered column) or use the global [__Clear All Filters__ button](#clear-all-filters).
```

Comment:
I'm not sure if it is the correct place, but it is worth mentioning that with this three-dots menu users are able to select which columns to see - just click on the three dots and then on the Columns in the popup - you'll have checkboxes for each available column and you can select which one you want to see.